niceman/tests/utils.py

```diff
@@ -382,18 +382,25 @@ def newfunc(*args, **kwargs):
     return newfunc
 
 
-def skip_ssh(func):
+def skip_ssh(func=None):
     """Skips SSH tests if on windows or if environment variable
     NICEMAN_TESTS_SSH was not set
     """
-    @wraps(func)
-    def newfunc(*args, **kwargs):
-        if on_windows:
-            raise SkipTest("SSH currently not available on windows.")
+
+    def check_and_raise():
         if not os.environ.get('NICEMAN_TESTS_SSH'):
             raise SkipTest("Run this test by setting NICEMAN_TESTS_SSH")
-        return func(*args, **kwargs)
-    return newfunc
+
+    if func:
+        @wraps(func)
+        def newfunc(*args, **kwargs):
+            if on_windows:
+                raise SkipTest("SSH currently not available on windows.")
+            check_and_raise()
+            return func(*args, **kwargs)
+        return newfunc
```

Comment:
now that there is no decorator, this code isn't tested ... suboptimal. ideally there should be a test (with `setup_docker`/`setup_ssh` fixture) which has inside another function which is `@skip_ssh` decorated, and calls it and verifies that it returns desired result.  This way we would assure at least that decorator works somewhat